tests/create_guardduty_member/main.tf

```
@@ -1,37 +1,42 @@
+# The provider account for the GuardDuty member account
 provider "aws" {
   region  = "us-east-1"
-  profile = "resource-member"
+  profile = "plus3it-member" # Profile must exist in your .aws/config
```

I want to follow the convention of the aws provider here...

```suggestion
  profile = "aws" # Profile must exist in your .aws/config
```

---

I used these profile names to make it clear which account was being used for what.  it was confusing for me to understand this using the more generic names.  I am trying to make this easier for the next person to understand.  I'll make this change if you really want me to, but I feel like we will lose some understandability.

---

Change made.  I would like to read about the test framework of the terraform aws provider.  I googled it but did not find much.  Can you please suggest a resource?